src/components/Card.js

```diff
@@ -64,11 +75,24 @@ export default class Card {
     return this._cardElement;
   };
 
+  delete() {
+    evt.target.closest(this._cardSelector).remove();
```

Comment:
Please, implement the `delete` method correctly, to remove the card element from the DOM

```js
  delete() {
    this._cardElement.remove();
  }
```